src/role-access/components/case-roles-table/case-roles-table.component.html

```diff
@@ -27,7 +27,7 @@
     </tr>
     <tr *ngIf="item.open">
       <td colspan="6" class="right">
-        <a [href]="action.id" *ngFor="let action of item.actions">{{action.title}}</a>
+        <a [href]="getActionHref(action, item)" *ngFor="let action of item.actions">{{action.title}}</a>
```

Comment:
for the reallocate role, I have amened the path in this way. See:
https://github.com/hmcts/rpx-xui-webapp/pull/1265/files#diff-4df32a271e33ce132fdd31c144331f65a3af61e1beb8c556b5ef337c4d8270b7

We will have conflicts here if we don't have the same path pattern.

Or should we split this actions(replace ngFor with single action item) as we are going to different path.

Reply:
as discuss this will leave with me